cloud/resource_pulsar_instance.go

```diff
@@ -146,9 +153,9 @@ func resourcePulsarInstanceCreate(ctx context.Context, d *schema.ResourceData, m
 			PoolRef:          poolRef,
 		},
 	}
-	if t == "serverless" {
```

Comment:
Why we remove the serverless support

Reply:
Now that it uses the `spec.type: serverless`, we no longer need this annotation